stylesheets/_component.select2.scss

```diff
@@ -76,12 +77,13 @@
     background-color: white;
     border: 1px solid $input-border;
     border-radius: 4px;
+    box-shadow: 0 5px 10px rgba(0,0,0,.4);
     box-sizing: border-box;
     display: block;
     position: absolute;
     left: -100000px;
     width: 100%;
-    z-index: 1051;
+    z-index: $zindex-modal;
```

Comment:
This would mean select2 dropsdowns appear above the secondary navigation flyout. Perhaps `$zindex-dropdown` would be more appropriate?

Reply:
@jamesjacobs 

When a select2 is in a modal, the dropdown will not be visible unless `$zindex-modal` is used.

When a select2 is inside a modal, the `.select2-dropdown` element is at the end of the DOM so it's not possible to change this to `.modal .select2-dropdown`

Select2 elements are closed when clicking outside the dropdown, which would make it extremely difficult for a dropdown to be above the navigation.

Reply:
Good stuff, move along!